'goober-autoprefixer@latest' is not in the npm registry.

So I was about to use the autoprefixer so i don't do extra steps with postcss and...

```
npm install goober-autoprefixer

npm ERR! 404 Not Found - GET https://registry.npmjs.org/goober-autoprefixer - Not found
npm ERR! 404  'goober-autoprefixer@latest' is not in the npm registry.
```

I checked npmjs.com and it turns out it really isn't out there.

--- a/packages/gatsby-plugin-goober/package.json
+++ b/packages/gatsby-plugin-goober/package.json
@@ -1,6 +1,6 @@
 {
     "name": "gatsby-plugin-goober",
-    "version": "1.0.0",
+    "version": "2.0.0",
     "description": "Gatsby plugin to add support for goober",
     "main": "index.js",
     "scripts": {},

--- a/packages/gatsby-plugin-goober/package.json
+++ b/packages/gatsby-plugin-goober/package.json
@@ -23,7 +23,7 @@
     "dependencies": {},
     "devDependencies": {},
     "peerDependencies": {
-        "goober": "^1.6.1",
+        "goober": "^2.0.0",
         "gatsby": "^2.10.0"
     },
     "repository": {

--- a/packages/goober-autoprefixer/package.json
+++ b/packages/goober-autoprefixer/package.json
@@ -6,5 +6,9 @@
   "umd:main": "./dist/goober-autoprefixer.umd.js",
   "source": "./src/index.js",
   "unpkg": "./dist/goober-autoprefixer.umd.js",
-  "types": "./autoprefixer.d.ts"
+  "types": "./autoprefixer.d.ts",
+  "author": {
+    "name": "Jovi De Croock",
+    "url": "https://www.jovidecroock.com/"
+  }
 }

